desc/compute/_profiles.py

```diff
@@ -701,7 +884,7 @@ def _iota_num_r(params, transforms, profiles, data, **kwargs):
             / data["psi_r"] ** 2,
             lambda: (
                 profiles["current"].compute(params["c_l"], dr=2) * data["psi_rr"]
-                - current_r * data["psi_rrr"]
+                - current_r * data["psi_rrr"]  # FIXME: psi_rrr=0
```

Comment:
`psi_rrr` (and higher derivatives) are always 0. Can we remove all references to them from the compute functions to simplify the code?